lib/australia/postcode.rb

```diff
@@ -33,15 +33,21 @@ def coords
     # @return [Float]
     def distance(other)
       earth_radius = 6371
-      Δlat = radians(other.latitude - latitude)
-      Δlong = radians(other.longitude - longitude)
-      a = sin(Δlat / 2) * sin(Δlat / 2) +
+      lat = radians(other.latitude - latitude)
```

Comment:
NIT: `delta_` / `change_in_` / `_difference`?

Reply:
I like the delta prefix. Will action when I have capacity to return to this.